src/tests/ParamTest.cs

```diff
@@ -1,149 +1,65 @@
 using Helium;
 using System;
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Threading.Tasks;
 using Xunit;
 
 namespace UnitTests
 {
     public class ParamTest
     {
         [Fact]
-        public void NoParamsTest()
+        public async Task NoParamsTest()
         {
             string[] args = Array.Empty<string>();
 
-            bool flag = App.ProcessArgs(args, out string kvUrl, out string authType, out bool helpFlag);
+            int i = await App.Main(args);
 
-            Assert.True(flag);
-            Assert.True(helpFlag);
-            Assert.Null(kvUrl);
-            Assert.Null(authType);
-        }
+            Assert.Equal(1, i);
 
```

Comment:
(just curious) why does this return 1?

Reply:
any non-zero return is an error to the shell. should probably assert not zero instead.